src/compiler/factory/nodeFactory.ts

```diff
@@ -5499,6 +5500,10 @@ namespace ts {
             );
         }
 
+        function createImportCall(argumentsList: readonly Expression[]) {

```

Comment:
I believe @rbuckton wanted this to be named `createImportExpression` (as far as I understood him correclty).

Reply:
I'll wait rbuckton to confirm this change

Reply:
My comment wasn't about renaming `createImportCall`, but whether we should have an `ImportExpression` for the `import` keyword, similar to how `SuperExpression` represents the `super` keyword, even though `super` isn't actually an expression. In a similar vein, we could have a `createImport()` method on `NodeFactory` similar to the `createSuper()` method. Then you could invoke `createCallExpression(createImport(), ...)` to create an `ImportCall`, just like you can invoke `createCallExpression(createSuper(), ...)` to create a `SuperCall`.

Reply:
🤔 I have to say `SuperExpression` is strange

Reply:
I think it would be better to change the AST type to accept specific keywords so that we can get type safety from it.

Reply:
> I think it would be better to change the AST type to accept specific keywords so that we can get type safety from it.

How do you think @rbuckton 👀